tests/r7rs/r7rs-tests.scm

```diff
@@ -505,22 +512,24 @@
       ((underscore2 (a _) ...) 42)))
   (test 42 (underscore2 (1 2))))
 
-(define-syntax count-to-2
+; TODO(higepon): duplicate pattern variable.
```

Comment:
Perhaps `_` is not recognized as `_` because it's not bound and/or exported.

Reply:
Thanks. I'll look into this later.